src/AutoRest.CSharp/Common/Input/Inputs.cs

```diff
@@ -31,7 +31,7 @@ internal record InputOAuth2Auth(IReadOnlyCollection<string> Scopes)
         public InputOAuth2Auth() : this(Array.Empty<string>()) {}
     }
 
-    internal record InputClient(string Name, string Description, IReadOnlyList<InputOperation> Operations)
+    internal record InputClient(string Name, string Description, IReadOnlyList<InputOperation> Operations, bool Creatable, IReadOnlyList<InputParameter> Parameters, string? Parent)
```

Comment:
```suggestion
    internal record InputClient(string Name, string Description, IReadOnlyList<InputOperation> Operations, bool Constructable, IReadOnlyList<InputParameter> ConstructParameters, string? Parent)
```

Reply:
Thanks @archerzz . Those parameter name are consistent with cadl definition. and it is not mean constructor parameters, it means optional parameters. The meaning of those part have not finalized yet. When cadl finalize those parameter name, and its exact meaning, we will update then if needed. By now, we will leave them as now.